homeassistant/components/google_assistant/trait.py

```diff
@@ -678,6 +679,52 @@ async def _execute_cover(self, command, data, params, challenge):
             )
 
 
+@register_trait
+class TemperatureControlTrait(_Trait):
```

Comment:
Isn't it confusing to name this `TemperatureControlTrait` when it's read only, maybe `TemperatureStateTrait` or `TemperatureSensorTrait` would be clearer?

Reply:
It's the google name for this trait : https://developers.google.com/assistant/smarthome/traits/temperaturecontrol

Reply:
Yes, but the newly added `TemperatureControlTrait` sets `queryOnlyTemperatureSetting = True`, meaning it doesn't support any controlling of the temperature.
Maybe name it `QueryOnlyTemperatureControlTrait`?